.github/workflows/e2e-main.yml

```diff
@@ -9,7 +9,7 @@ on:
       - "docs/**"
       - "**.md"
       - ".circleci/**"
-      - "**unit.spec.js"
```

Comment:
Although `**unit` part works as expected even without a dot `.` (`**` replaces ANY character), I've added it back for the sake of being explicit